tools/pipelines/templates/include-test-real-service.yml

```diff
@@ -232,13 +230,39 @@ jobs:
           workingFile: ${{ parameters.testWorkspace }}/.npmrc
           customEndpoint: 'Office Packages - Readonly'
 
-      # Install test and logger package
+      # Install logger package
+      - task: Npm@1
+        displayName: 'npm install'
```

Comment:
I had to do this split since we had some issues with internal package (aria logger) not being present on our feeds. We likely do not need this change. Can you please try with this file reverted? If install goes through, we are fine and do not need this.

Reply:
reverted